glib/src/value.rs

```diff
@@ -205,6 +205,23 @@ pub unsafe trait FromValue<'a>: Sized {
     unsafe fn from_value(value: &'a Value) -> Self;
 }
 
+/// Similar to `FromValue` but works for owned `Value.
+///
+/// ```compile_fail
+/// let x = "gtk-rs".to_value();
```

Comment:
Maybe add some comments like "This example cannot compile because `&str` is a temporary value" or something along the line. And please also add a working code example (with `String` for example). So both are always checked.